canary/apps/react/vite3/src/App.tsx

```diff
@@ -0,0 +1,27 @@
+import {
+  AmplifyProvider,
+  Button,
+  Card,
+  Heading,
+  withAuthenticator,
+  useAuthenticator,
+} from '@aws-amplify/ui-react';
+import '@aws-amplify/ui-react/styles.css';
+import Amplify from 'aws-amplify';
+import aws_exports from '../../../../environments/auth-with-email/src/aws-exports';
+Amplify.configure(aws_exports);
+
+function App() {
+  const { signOut } = useAuthenticator();
+  return (
+    <AmplifyProvider colorMode="dark">
```

Comment:
Maybe just a nit: should we update these to ThemeProvider?

Reply:
Ok will do!

Reply:
148cc9b9b44b35afd0a0bfb6d9ff20cd2f44151b